### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Saiv46**
Added some info about peering protocols/options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,14 +36,24 @@ network.
...
-the peering connection inside a regular TLS session, which can help in some cases
-where firewalls or deep packet inspection may identify or block regular Yggdrasil
-peering traffic.
+Currently Yggdrasil only operates as overlay network, so it requires underlaying networking.
+As for 0.4.5, following protocols supoorted for peering:
 
-TLS public peers are identified by the prefix `tls://` instead of `tcp://`. 
+- Plain TCP (`tcp://host:port`)
+- TLS (`tls://host:port`)
+- TCP over SOCKS5 (`socks://user:pass@proxy/host:port`)
```
Comment (cathugger) [inline README.md:46]: not TLS over SOCKS5? (i may be misremembering this one)

Comment (cathugger) [reply README.md:46]: can't exactly find it myself right now so probably just misremembering.

Comment (Saiv46) [reply README.md:46]: I looked up on source code before making this PR, and there's no TLS over SOCKS5

**Comment by Saiv46** [comment]
Superseded by #523

**Pull request closed without merging**